### DISCUSSION TITLE
在vuepress 里面如何使用unocss

### DISCUSSION THREAD

**Discussion by ALiangTech** [vuepress/core#1402] [Q&A] [answered]
import 'uno.css' 没有入口文件 在enhanceApp.js 引入后没啥作用
unocss 不起作用的

**Comment by meteorlxy** [comment]
enhanceApp.js 是 v1 的

https://v2.vuepress.vuejs.org/zh/guide/migration.html#%E7%BA%A6%E5%AE%9A%E6%96%87%E4%BB%B6%E5%8F%98%E6%9B%B4

**Reply by ALiangTech** [reply]
好滴 我看看

**Comment by ALiangTech** [comment, marked as answer]
通过脚本向app.js里面插入import 'uno.css'

**Reply by gjfLeo** [reply]
请问下，这是如何实现的？
